.github/workflows/test.yml

```
@@ -0,0 +1,42 @@
+name: Tests
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+
+jobs:
+  test:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 12
+          - 14
+          - 16
+
```

```suggestion
          - 18
```

might want to add 18 as well :)